Core/models.py

```diff
@@ -1,3 +1,87 @@
 from django.db import models
+from django.core.validators import RegexValidator
 
-# Create your models here.
+
+class PhonePrefix(models.Model): # This model might be located in diffirent app 
+
+    # PREFIXES = (
+    #     ('+99455','+99455'),
+    #     ('+99450','+99450'),
+    #     ('+99451','+99451'),
+    #     ('+99470','+99470'),
+    #     ('+99477','+99477'),
+    #     ('+99499','+99499'),
+    # ) 
+    # since task requires not choice field, I decided to keep it as Charfield. 
+    # I think in that way any user could not add new prefixes (they just could use existing ones)  
+
+    prefix = models.CharField(max_length=6)
+
+    def __str__(self):
+        return self.prefix 
+
+    class Meta:
+        verbose_name_plural = "Phone Prefixes"
+
+class WareHouse(models.Model): # This model might be located in diffirent app 
+
+    name = models.CharField(max_length=32)
+
+    def __str__(self):
+        return self.name
+    
+    class Meta:
+        verbose_name_plural = "Warehouses"
+
+
+class Currency(models.Model):
+
+    name = models.CharField(max_length=32)
+    sign = models.CharField(max_length=1)
+    rate = models.FloatField()
+
+    def __str__(self):
+        return f'{self.name} {self.id}' 
+
+
+    class Meta:
+        verbose_name_plural = "Currencies"
+
+
+
+class Country(models.Model):
+
+    name = models.CharField(max_length=32)
+    is_active = models.BooleanField(default=True)
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name_plural = "Countries"
+
+
+class SingletonModel(models.Model):
```

Comment:
It would be better, If you did not put this into model file, as it is not db model. Plus, there should be no db entry for this, as this is just utility class